<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScripterMainClass</class>
 <widget class="QMainWindow" name="ScripterMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>注册码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="unBindBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>解绑按钮</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>充值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chongzhiLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chongzhiBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>充值按钮</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>版本：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="ScripterMain.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ScripterMainClass</receiver>
   <slot>enterBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enterBtnClicked()</slot>
 </slots>
</ui>
